packages/cdktf-cli/bin/cmds/ui/terraform-context.tsx

```diff
@@ -431,7 +448,10 @@ export const useTerraform = ({
 
   const execTerraformInit = async () => {
     try {
-      if (!terraform) throw new Error("Terraform is not initialized yet");
+      if (!terraform)
+        throw new Error(
+          "Terraform is not initialized yet. Please call synth() with loadExecutor: true first"
```

Comment:
I think this is a great internal error message, but if it makes its way to the user it could be really confusing. Is there something a user can do?

Reply:
I'm torn – a user should never see this message in the first place (same issue with the original error message as there is no `cdktf init` which the user could use to resolve Terraform not being initialized).

Reply:
I see your point and I always remember the anecdote my mom told me where in an important client meeting a very internal error message was displayed :D 

Maybe we can prefix it with ÌnternalError to indicate this is not a user-facing error? I'm fine with postponing this and making it a separate issue :)

Reply:
Good idea, prefixed it with "Internal error: "